go.mod

```diff
@@ -189,6 +189,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/nmstate/nmstate/rust/src/go/nmstate v0.0.0-20220811151154-801022633c42 // indirect
```

Comment:
It looks like `nmstate` was not vendored in:
```
 vendor/github.com/openshift/assisted-service/pkg/staticnetworkconfig/generator.go:13:2: cannot find package "github.com/openshift/installer/vendor/github.com/nmstate/nmstate/rust/src/go/nmstate" in:
	/go/src/github.com/openshift/installer/vendor/github.com/nmstate/nmstate/rust/src/go/nmstate 
```

Reply:
My bad, forgot to `git add`.
I don't think this is gonna compile in CI though :(